### PR TITLE
VW-76240: Max Capacity changed byte to mb

### PR DESCRIPTION
**Pull request by vinodpatilVI**

### CODE REVIEW

File: internal/nutanix/storage_containers.go [hunk 1]
```diff
@@ -147,8 +148,25 @@ func (e *StorageContainerExporter) Collect(ch chan<- prometheus.Metric) {
 		key := KEY_STORAGE_CONTAINER_PROPERTIES
 		var property_values []string
 		for _, property := range e.properties {
-			val := fmt.Sprintf("%v", ent[property])
-			property_values = append(property_values, val)
+			var val string = ""
+			// format properties
+			switch property {
+			case "max_capacity":
```
Comment (hgokhale) [inline internal/nutanix/storage_containers.go:154]: We should change the name to max_capacity_mb

Comment (hgokhale) [reply internal/nutanix/storage_containers.go:154]: This also needs to be fixed in rules.yaml and the schema in capi

**Review by hgokhale — CHANGES_REQUESTED** [review]

File: internal/nutanix/storage_containers.go [hunk 2]
```diff
@@ -147,7 +149,24 @@ func (e *StorageContainerExporter) Collect(ch chan<- prometheus.Metric) {
 		key := KEY_STORAGE_CONTAINER_PROPERTIES
 		var property_values []string
 		for _, property := range e.properties {
-			val := fmt.Sprintf("%v", ent[property])
+			var val string = ""
+			// format properties
+			switch property {
+			case "max_capacity_mb":
+				propname := "max_capacity_mb"
```
Comment (hgokhale) [inline internal/nutanix/storage_containers.go:156]: This seems incorrect... Should this be just "max_capacity'? Please test the changes throughly

Comment (vinodpatilVI) [reply internal/nutanix/storage_containers.go]: push again

Comment (hgokhale) [reply internal/nutanix/storage_containers.go]: Would you share the output... Output should have max_capacity_mb right?

**Review by hgokhale — APPROVED** [review]

**Pull request merged**